curriculum/challenges/english/25-front-end-development/lab-celsius-to-fahrenheit-converter/56533eb9ac21ba0edf2244b3.md

```diff
@@ -71,6 +65,12 @@ assert.strictEqual(convertCtoF(20), 68);
 assert.strictEqual(convertCtoF(30), 86);
 ```
 
+You should only pass a numeric argument to the `convertCtoF` function.
+
+```js
+assert.match(code, /convertCtoF\s*\(\s*-?\d+(\.\d+)?\s*\)/);
+```
```

Comment:
what is this testing? I am confused on what this is doing

Reply:
This function *looks* like it's checking to see if the camper called the function in their solution. But I don't think it should be a requirement.

Reply:
then you need to remove the test, this fails if the camper does not call the function